content/applications/finance/payment_acquirers/ogone.rst

```diff
@@ -5,84 +5,105 @@ Ogone
-:menuselection:`Configuration --> Technical Information --> Transaction feedback --> All transaction
-submission modes`, and get or generate your **API Key** and **Client Key**. Be careful to copy your
-API key as you'll not be allowed to get it later without generating a new one.
+You are now required to generate a **SHA-IN** passphrase. **SHA-IN** and **SHA-OUT** passphrases are
+used to digitally sign the transaction requests and responses between Odoo and Ogone. By using these
+secret passphrases and the **SHA-1** algorithm, both systems can ensure that the information they
+receive from the other was not altered or tampered with.
 
 .. important::
-   If you are trying Ogone as a test, with the Test Account, change the **State** to *Test Mode*. We
-   recommend doing this on a test Odoo database, rather than on your main database.
+   If you need to use another algorithm, such as **SHA-256** or **SHA-512**, within Odoo, activate
+   the :ref:`developer mode <developer-mode>` and go to :menuselection:`General Settings -->
+   Technical --> System Parameters`. From here, search for `payment_ogone.hash.function` and
+   change the value line to the desired algorithm (`SHA-256` or `SHA-512`).
+
+Your **SHA-IN** and **SHA-OUT** passphrases should be different, and between 16 and 32 characters
+long. Make sure to use the same **SHA-IN** and **SHA-OUT** passphrases throughout the entire Ogone
+configuration, as Odoo only allows a single **SHA-IN** and single **SHA-OUT** passphrase. In the
+:guilabel:`DirectLink and Batch` section of the screen, add the same **SHA-IN** passphrase as the
+previous field. You can leave the IP address field blank.
+
+When done, head to :menuselection:`Configuration --> Technical Information --> Transaction Feedback`
+and check the following options:
+
+- :guilabel:`I would like to receive transaction feedback parameters on the redirection URLs`:
+  should be checked.
+- :guilabel:`Direct HTTP server-to-server request`: should to be set to *Online*.
+- Both **URL** fields should contain the same following URL, with `example` replaced by your
+  database: `https://<example>/payment/ogone/accept`.
+
+- :guilabel:`Dynamic eCommerce Parameters` should contain contain the following values:
+  `ALIAS`, `AMOUNT`, `CARDNO`, `CN`, `CURRENCY`, `IP`, `NCERROR` `ORDERID`, `PAYID`, `PM`,
+  `STATUS`, `TRXDATE`. Other parameters can be included (if you have another integration with Ogone
+  that requires them), but are not advised.
+- The :guilabel:`URL` fields for :guilabel:`HTTP redirection in the browser` can be left empty, as
+  Odoo will specify these URLs for every transaction request.
+- In the :guilabel:`All transaction submission modes` section, fill-out **SHA-OUT** passphrase and
+  disable `HTTP request for status change`.
+
+.. note::
+   If you use tokenization (allowing users to save their cards), go to :menuselection:`Configuration
+   --> Alias` and set up the tokenization process according to your needs.
```

Comment:
Isn't that a bit vague? Requiring the admin to enable tokenization on Ogone's side would allow you to remove the note at the very end of the page.